lib/src/Uncertainty/Distribution/openturns/MarginalDistribution.hxx

```diff
@@ -177,6 +201,17 @@ private:
   /** The upper bound of the underlying distribution */
   Point upperBound_;
 
+  /** Flag to tell if the computations have to be done using the CDF or the PDF
+      of the underlying distribution */
+  Bool usePDF_ = ResourceMap::GetAsBool("MarginalDistribution-UsePDF");
```

Comment:
must be serialized

Reply:
@jschueller Which means?

Reply:
oh, it must appear in the save/load methods

Reply:
@jschueller Ah, I forgot it! I was thinking it was about the way it is initialized. Thks.